frontend/cypress/pages/metrics/metrics.ts

```diff
@@ -174,6 +175,10 @@ class Metrics {
     this.pipelineRemoveButton().click()
   }
 
+  closeOptions() {
+    cy.get('[data-test-id="Header"]').click()
```

Comment:
`cy.get('[data-test-id="Header"]')` extract it as a const/variable, not use it directly, follow e2e pattern`POM`